payments-core/src/main/java/com/stripe/android/paymentsheet/BaseAddPaymentMethodFragment.kt

```diff
@@ -160,7 +162,13 @@ internal abstract class BaseAddPaymentMethodFragment(
         addSaveForFutureUseArguments(
             args,
             isCustomer = sheetViewModel.customerConfig != null,
-            isSetupIntent = sheetViewModel.stripeIntent.value is SetupIntent
+            isSetupIntent = (
```

Comment:
`isSetupIntent` means something else now

Reply:
ack this, we should probably change the param name to something other than `isSetupIntent`